src/internal/operations/test/exchange_test.go

```diff
@@ -0,0 +1,767 @@
+package test_test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/alcionai/clues"
+	"github.com/microsoftgraph/msgraph-sdk-go/users"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"golang.org/x/exp/maps"
+
+	"github.com/alcionai/corso/src/internal/common/dttm"
+	inMock "github.com/alcionai/corso/src/internal/common/idname/mock"
+	"github.com/alcionai/corso/src/internal/common/ptr"
+	"github.com/alcionai/corso/src/internal/events"
+	evmock "github.com/alcionai/corso/src/internal/events/mock"
+	"github.com/alcionai/corso/src/internal/m365/exchange"
+	exchMock "github.com/alcionai/corso/src/internal/m365/exchange/mock"
+	exchTD "github.com/alcionai/corso/src/internal/m365/exchange/testdata"
+	"github.com/alcionai/corso/src/internal/m365/resource"
+	"github.com/alcionai/corso/src/internal/tester"
+	"github.com/alcionai/corso/src/internal/version"
+	deeTD "github.com/alcionai/corso/src/pkg/backup/details/testdata"
+	"github.com/alcionai/corso/src/pkg/control"
+	"github.com/alcionai/corso/src/pkg/fault"
+	"github.com/alcionai/corso/src/pkg/path"
+	"github.com/alcionai/corso/src/pkg/selectors"
+	"github.com/alcionai/corso/src/pkg/services/m365/api"
+)
+
+type ExchangeBackupIntgSuite struct {
+	tester.Suite
+	its intgTesterSetup
+}
+
+func TestExchangeBackupIntgSuite(t *testing.T) {
+	suite.Run(t, &ExchangeBackupIntgSuite{
+		Suite: tester.NewIntegrationSuite(
+			t,
+			[][]string{tester.M365AcctCredEnvs, tester.AWSStorageCredEnvs}),
+	})
+}
+
+func (suite *ExchangeBackupIntgSuite) SetupSuite() {
+	suite.its = newIntegrationTesterSetup(suite.T())
+}
+
+// TestBackup_Run ensures that Integration Testing works
+// for the following scopes: Contacts, Events, and Mail
+func (suite *ExchangeBackupIntgSuite) TestBackup_Run_exchange() {
```

Comment:
Technically not, I suppose?  But I don't see any harm in it sticking around.  It's nice to be able to ensure basic assertions outside of the extra behavior.  

It's not entirely replaced, for that matter.  One of the cases it does cover that the more detailed tests do not is events.  Since our test users' event calendars regularly break, we don't currently include that data in incremental testing.  This test gives us some regression coverage there.